README

```diff
@@ -404,6 +413,29 @@ defined in the authorization mapping file. If during an authentication attempt
 a connected device is removed or a new device is plugged in, the authentication
 restarts from the top of the list.
 
+[[confFile]]
+== Configuration file
+
+A configuration file can be used to set the default
+<<moduleArguments,module arguments>>.
+
+The file has a `name = value` format, with comments starting with the `#`
```

Comment:
This section reads like a list, but is not a list, should we rephrase or add bullet points?

(same comment for man pages.   as an additional follow up, we might want to consider a `pam_u2f.conf.5`)